etc/kayobe/ansible/scripts/seed-ubuntu-upgrade.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/bash
+
+set -e
+
+if [[ ! $1 ]]; then
+    echo "Usage: seed-ubuntu-upgrade.sh <seed-hostname>"
+    exit 2
+fi
```

Comment:
There should only be a single seed host

Reply:
But do they always have same host name?

Reply:
It doesn't matter - plays target the `seed` group

Reply:
We don't need the hostname, because we never need to limit it

Reply:
I see